flow/__main__.py

```diff
@@ -46,6 +49,43 @@ def main_init(args):
         )
 
 
+def main_template(args):
+    """Print out general information about template subcommand."""
+    print("flow template does nothing. Use of the available subcommands [create,].")
+
+
+def main_template_create(args):
+    """Create custom template based on a specified or detected environment."""
+    if args.name is None:
+        env = environment.get_environment()
+        if env is environment.StandardEnvironment:
+            extend_template = "base_script.sh"
+        else:
+            extend_template = env.template
+    else:
+        extend_template = args.name
+
+    project = get_project()
+    os.makedirs(project.fn("templates"), exist_ok=True)
+
+    # grab and render custom template
+    jinja_env = jinja2.Environment(loader=jinja2.PackageLoader("flow"))
+    custom_template_template = jinja_env.get_template("custom.sh")
+    new_template = custom_template_template.render(extend_template=extend_template)
```

Comment:
Why is this a "template template"?
```suggestion
    custom_template = jinja_env.get_template("custom.sh")
    new_template = custom_template.render(extend_template=extend_template)
```